addons/promote/model-java/src/main/java/org/commonjava/indy/promote/model/PathsPromoteRequest.java

```diff
@@ -52,6 +52,9 @@
     @ApiModelProperty( value="Fire events, e.g. PromoteCompleteEvent" )
     private boolean fireEvents;
 
+    @ApiModelProperty( value="Whether to fail the promotion if path exists in target repo" )
+    private boolean failWhenExists;
```

Comment:
I thought we agreed that this is not needed, because it can be done by no-pre-existing-paths.groovy validation rule.

Reply:
Ok. It seems the validation will fail the promotion and we won't need to check it here.

Reply:
Maybe I keep this flag for internal use, e.g., rollback. I set default to true. When doing rollback, set it to false so the unpurged files won't bring any trouble.

Reply:
If you're using it for internal purposes, can we keep it out of the public api though? I think that might reduce confusion...